Redesign

Thunae opened the pull request:
No more bootstrap. Website is completely redone.
Learned how to use CSS animations and transitions (can't be seen in the pictures :( )
Better visual design
Need to add some content to pages, there are some compatibility issue with different device sizes

![Screenshot 2020-06-01 at 12 12 00 PM](https://user-images.githubusercontent.com/51490436/83429338-ea3cd800-a401-11ea-87da-f861450feced.png)
![Screenshot 2020-06-01 at 12 12 20 PM](https://user-images.githubusercontent.com/51490436/83429350-edd05f00-a401-11ea-897f-c05e4197df7b.png)
![Screenshot 2020-06-01 at 12 12 37 PM](https://user-images.githubusercontent.com/51490436/83429351-edd05f00-a401-11ea-82cf-3256641e169b.png)
![Screenshot 2020-06-01 at 12 12 55 PM](https://user-images.githubusercontent.com/51490436/83429352-ee68f580-a401-11ea-97a9-b3ce2af92b25.png)
![Screenshot 2020-06-01 at 12 13 07 PM](https://user-images.githubusercontent.com/51490436/83429356-ef018c00-a401-11ea-8c59-c0adb2453318.png)
![Screenshot 2020-06-01 at 12 13 20 PM](https://user-images.githubusercontent.com/51490436/83429359-ef9a2280-a401-11ea-95d6-93c7b3c18608.png)
![Screenshot 2020-06-01 at 12 13 39 PM](https://user-images.githubusercontent.com/51490436/83429362-ef9a2280-a401-11ea-9d63-6ae39078d3b9.png)

lcatherine commented:
Looks good :) 

Two things:
1) Code-wise please refer to the style guide we linked in the Pod chat and fix the inconsistent styling.
2) Commit-wise please squash your commits before submitting and be sure to change the commit message as needed to encapsulate all the changes.

Thunae commented:
I think I squashed the commits, and I also fixed some of the styling issues

Thunae commented:
> Looks good :)
> 
> Two things:
> 
> 1. Code-wise please refer to the style guide we linked in the Pod chat and fix the inconsistent styling.
> 2. Commit-wise please squash your commits before submitting and be sure to change the commit message as needed to encapsulate all the changes.

I think I squashed the commits, and I also fixed some of the styling issues just now

lyze commented on portfolio/src/main/webapp/skills.html:
```diff
@@ -0,0 +1,42 @@
+      <link rel="stylesheet" href = "style/format.css">
+   </head>
+   <body class="menu-page page">
+      <div class = "skills-background">
```
there are still some formatting inconsistencies here and also in other HTML files :(

Thunae replied:
I believe the formatting is fixed now

lyze replied:
I think the formatting is still off in a few files :(

Thunae replied:
Just formatted with the online formatter

lyze commented on portfolio/src/main/webapp/script.js:
```diff
@@ -26,6 +26,26 @@ function addRandomGreeting() {
+function fadeOut(){
+    var page = document.getElementsByClassName("page");
+    page[0].style.opacity = '0';
+    setTimeout(function(){window.location.href = "./main.html";}, 1000);
```
Can we make this an arrow too? Since we use arrows below.

Thunae replied:
I combined these into a single line with parallel style to above (.then), but I didn't use arrow functions

lyze replied:
i find the .then one-liner harder to read now.

I was just suggesting `setTimeout(() => { window.location.href = ... })`

Thunae replied:
Changed to your suggestion

Thunae commented on portfolio/src/main/webapp/about.html:
```diff
@@ -1,47 +1,39 @@
 <!DOCTYPE HTML>
-<html lang = "en">
```
changed
